lib/src/chart/line_chart/line_chart_data.dart

```diff
@@ -572,17 +572,17 @@ bool showAllSpotsBelowLine(FlSpot spot) {
   return true;
 }
 
-/// Holds data for showing dots on the bar line.
-class FlDotData {
+typedef GetDotColorCallback = Color Function(FlSpot);
+Color _defaultGetDotColor(FlSpot _) => Colors.blue;
 
+/// This class holds data about drawing spot dots on the drawing bar line.
+class FlDotData {
   /// Determines show or hide all dots.
   final bool show;
-
-  /// Determines color of showing dots.
```

Comment:
As you may know, we've started to write documentation for all public classes and functions,
You've removed these comments, please take them back, and write documentation for added `GetDotColorCallback`, `strokeWidth`, `getStrokeColor`, and `getDotColor` too.

Reply:
Fixed